app/controllers/article_controller.rb

```diff
@@ -2,5 +2,80 @@
 
 # ArticleController is the controller for Article Search
 class ArticleController < ApplicationController
+  include Blacklight::Configurable
+
+  before_action :eds_init, only: %i[index show]
+  # TODO: probably need to move this into an Eds::SearchService initializer
+  def eds_init
+    session['guest'] = false # TODO: hardcoded to bypass auth
+    session['eds_session_token'] =
+      EBSCO::EDS::Session.new(
+        guest: false, # TODO: hardcoded to bypass auth
```

Comment:
Wondering if we should make this true by default, even for our initial implementation.  Currently, anybody on campus could theoretically access this site if they knew about it, and could potentially access metadata if items they are technically not supposed to be able to.